zebra-rpc/src/server/http_request_compatibility.rs

```diff
@@ -103,11 +103,13 @@ impl FixHttpRequestMiddleware {
             .replace(", \"jsonrpc\": \"1.0\"", "")
     }
 
-    /// If the `content-type` HTTP header is not present,
-    /// add an `application/json` content type header.
-    pub fn add_missing_content_type_header(headers: &mut hyper::header::HeaderMap) {
-        headers
-            .entry(hyper::header::CONTENT_TYPE)
-            .or_insert(hyper::header::HeaderValue::from_static("application/json"));
+    /// Ignore client supplied `content-type` HTTP header if any and always send requests with `application/json`.
+    ///
+    /// `application/json` is the only `content-type` accepted by the Zebra rpc endpoint.
+    pub fn add_content_type_header(headers: &mut hyper::header::HeaderMap) {
```

Comment:
Nitpick: this is actually replacing the header, not adding an extra header with a different value
```suggestion
    pub fn replace_content_type_header(headers: &mut hyper::header::HeaderMap) {
```

Reply:
This is an outdated comment.